h/static/styles/partials-v2/_search.scss

```diff
@@ -225,15 +225,27 @@
                  // work).
 }
 
+$stats-icon-column-width: 20px;
```

Comment:
Ah, cool. I probably should have used variables in some of the CSS I've written recently :)